tensorboard/plugins/core/core_plugin.py

```diff
@@ -90,10 +91,18 @@ def get_plugin_apps(self):
             "/histograms": self._redirect_to_index,
             "/images": self._redirect_to_index,
         }
-        apps.update(self.get_resource_apps())
+        apps.update(self._static_handlers)
```

Comment:
why not just call the function here if `self._static_handlers` is not used in other places?

Reply:
Hmm good point, earlier I was going to need the property but now I don't.  Will inline.